mtags/src/main/scala-3/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -14,15 +16,21 @@ import dotty.tools.dotc.core.Definitions
 import dotty.tools.dotc.core.Flags
 import dotty.tools.dotc.core.Flags.*
 import dotty.tools.dotc.core.NameKinds.DefaultGetterName
+import dotty.tools.dotc.core.NameKinds.NameKind
 import dotty.tools.dotc.core.Names.Name
 import dotty.tools.dotc.core.StdNames
 import dotty.tools.dotc.core.SymDenotations.SymDenotation
 import dotty.tools.dotc.core.Symbols.NoSymbol
 import dotty.tools.dotc.core.Symbols.Symbol
 import dotty.tools.dotc.core.Types.Type
+import dotty.tools.dotc.util.SourceFile
 import dotty.tools.dotc.util.SourcePosition
+import dotty.tools.dotc.util.Spans.Span
+import org.eclipse.{lsp4j as l}
 
 object OverrideCompletions:
+  private val DefaultIndent = 2
```

Comment:
Are we able to get the indentation data from the scanner maybe? Not that important, just curious how does the presentation compiler can provide us with it.

Reply:
I couldn't figure out how to retrieve the indentation width from the presentation compiler/context, indentation will be calculated during scanner phase, but I couldn't find the API to retrieve it after the phase 🤔

Reply:
That might be :/ I didn't get any responses from the compiler team, so let's leave it as is.